### PR TITLE
feat(odd): protocol setup, instruments and lpc 

### PR DESCRIPTION
**Pull request by b-cooper**
# Overview

Add visuals and wire up protocol setup flow from start to finish. Create instruments step and
labware position check step which launches the flow.

# Changelog

- add Instruments setup step and wire it up to instruments and calibration data endpoints
- add LPC setup step and wire up to existing flow without applying new styles
- remove old non-hifi artifacts from robot dashboard.
- add loading shimmer to protocol setup

# Risk assessment
low

### CODE REVIEW

File: app/src/organisms/SendProtocolToOT3Slideout/index.tsx [hunk 1]
```diff
@@ -135,7 +135,7 @@ export function SendProtocolToOT3Slideout(
       }
       selectedRobot={selectedRobot}
       setSelectedRobot={setSelectedRobot}
-      showOT3Only
+      // showOT3Only
```
Comment (shlokamin) [inline app/src/organisms/SendProtocolToOT3Slideout/index.tsx]: leftover comment

File: robot-server/simulators/test.json [hunk 2]
```diff
@@ -8,14 +8,7 @@
...
-    "left": {
-      "model": "p10_single_v1",
-      "id": "123",
-      "max_volume": 10,
-      "name": "p10_single",
-      "tip_length": 0,
-      "channels": 1
-    }
+    "left": null
   },
```
Comment (shlokamin) [inline robot-server/simulators/test.json]: leftover simulator change

**Review by shlokamin — APPROVED** [review]
LGTM, thanks for all of the clean up!

File: app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+  getGripperDisplayName,
+  getPipetteNameSpecs,
+  PipetteName,
+  SINGLE_MOUNT_PIPETTES,
+} from '@opentrons/shared-data'
+
+import { SmallButton } from '../../atoms/buttons/OnDeviceDisplay'
+import { ChoosePipette } from '../PipetteWizardFlows/ChoosePipette'
+
+import type { Mount } from '../../redux/pipettes/types'
```
Comment (koji) [inline app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx]: probably we want to move this after @opentrons import statements

File: app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+): JSX.Element {
+  const { t } = useTranslation('protocol_setup')
+  const {
+    mount,
+    attachedInstrument,
+    speccedName,
+    attachedCalibrationData,
+  } = props
+
+  const [showChoosePipetteModal, setShowChoosePipetteModal] = React.useState(
```
Comment (koji) [inline app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx:71]:
```suggestion
  const [showChoosePipetteModal, setShowChoosePipetteModal] = React.useState<boolean>(
```

File: app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+        />
+      ) : null}
+    </>
+  )
+}
+
+const MountLabel = styled.p`
+  font-weight: ${TYPOGRAPHY.fontWeightSemiBold};
+  text-align: ${TYPOGRAPHY.textAlignLeft};
+  text-transform: ${TEXT_TRANSFORM_CAPITALIZE};
```
Comment (koji) [inline app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx]:
```suggestion
  text-transform: ${TYPOGRAPHY.textTransformCapitalize};
```

File: app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+  font-size: ${TYPOGRAPHY.fontSize22};
+  line-height: ${TYPOGRAPHY.lineHeight28};
+`
+
+const SpeccedInstrumentName = styled.p`
+  font-weight: ${TYPOGRAPHY.fontWeightRegular};
+  text-align: ${TYPOGRAPHY.textAlignLeft};
+  text-transform: ${TEXT_TRANSFORM_CAPITALIZE};
```
Comment (koji) [inline app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx]: same as the above

File: app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+  font-size: ${TYPOGRAPHY.fontSize22};
+  line-height: ${TYPOGRAPHY.lineHeight28};
+`
+
+const CalibrationStatus = styled.p`
+  font-weight: ${TYPOGRAPHY.fontWeightSemiBold};
+  text-align: ${TYPOGRAPHY.textAlignLeft};
+  text-transform: ${TEXT_TRANSFORM_CAPITALIZE};
```
Comment (koji) [inline app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx]: same as the above

Comment (koji) [reply app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx]: seems this is almost same as `MountLabel`.
maybe

```ts
styled(MountLabel)`
  color: ${({ color }) => color};
`
```

File: app/src/organisms/LabwarePositionCheck/JogToWell.tsx [hunk 8]
```diff
@@ -21,6 +22,7 @@ import {
   getVectorDifference,
   getVectorSum,
   PipetteName,
+  ODD_MEDIA_QUERY_SPECS,
```
Comment (koji) [inline app/src/organisms/LabwarePositionCheck/JogToWell.tsx]: Jethary moved this to `components/src/ui-style-constants/responsiveness.ts` recently

now this should be 

```ts
import { RESPONSIVENESS } from '@opentrons/components'

@media ${RESPONSIVENESS.touchscreenMediaQuerySpecs}
```

need to get merged recent edge

File: app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx (hunk 4 above)
Comment (b-cooper) [reply app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx:71]: TS can infer primitve `useState` types based on the initial value.

File: app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx (hunk 7 above)
Comment (b-cooper) [reply app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx]: yeah It is nearly identical.  Once we've gotten our text variations in order, ideally these would use the same grouping from a shared global abstraction (e.g. <StyledText as="p" fontWeight="semibold" />...). But for now, there's not much reason to combine these two, the naming clarifies exactly where they are used.

File: app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx [hunk 9]
```diff
@@ -408,3 +427,29 @@ export function ProtocolSetup(): JSX.Element {
...
+function ProtocolSetupSkeleton(props: ProtocolSetupSkeletonProps): JSX.Element {
+  return (
+    <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacingXXL}>
+      <Flex justifyContent={JUSTIFY_SPACE_BETWEEN}>
+        <Flex flexDirection={DIRECTION_COLUMN} gridGap="0.25rem">
+          <Skeleton height="2rem" width="7rem" backgroundSize="64rem" />
+          <Skeleton height="2rem" width="28rem" backgroundSize="64rem" />
+        </Flex>
+        <Flex gridGap={SPACING.spacing5}>
+          <CloseButton onClose={() => props.cancelAndClose()} />
```
Comment (koji) [inline app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx:443]: For consistency, probably we would like avoid `props.` I think we do 👇 if there is no specific reason.

```ts
function ProtocolSetupSkeleton({cancelAndClose}: ProtocolSetupSkeletonProps): JSX.Element {
or
const { cancelAndClose } = props
```

```suggestion
          <CloseButton onClose={() => cancelAndClose()} />
```

**Review by koji — APPROVED** [review]

**Pull request merged**